### ISSUE TITLE
Allows us to configure the refresh interval for /checks/status/

### ISSUE THREAD

**Issue by danemacmillan** [healthchecks/healthchecks#180]
Right now the default seems to be `3s` for the `/checks/status/` endpoint. I realize this will add greater stress to the hosted version, so perhaps include it on one of the paid plans. I'm paying for Standard right now.

I only recently needed this as I have some jobs that run sub-minute, so seeing the refresh more configurable would be nice.

Great service!

**Comment by cuu508** [comment]
Thanks for the idea!

The added load would be manageable -- as long as we're careful not to run 1s refresh indefinitely when somebody has a healthchecks.io tab open and goes on a vacation (and similar scenarios).

I'm guessing, with sub-minute pings you often see "XX seconds ago" in the last ping column, and the value is not steadily going up every second, but jumping by 3 every 3 seconds. Do you just want the counter to go up every second so it looks more smooth and realtime, or do you in fact need the extra precision?

If it's the former, we could also cheat a bit and extrapolate the values on client side, keeping the 3 second refresh interval.

**Comment by danemacmillan** [comment]
I would definitely want to have precision over the appearance. I actually think the last ping time should always just reflect that, and not fill in the gaps, so I'm against the idea of 1s updates just to look smooth.

I'd like to have it customizable, so, for example, if this were implemented, I would see the default refresh interval you set as `3s`, and then I could select another option, like `1s` or `5s` or `10s`, etc. This is why I would say push it onto a paid plan, specifically the low refresh times like `1s`. 

I have to admit, I have no immediate need for it. The moment I saw the `3s` refresh, the thought popped into my head. It was a knobs-and-dials moment for me. I recently added three checks that occur every 30 seconds, and I know as I integrate more checks into operations, I'll be adding more sub-minute checks.
